### DISCUSSION TITLE
Disable auto-collapse of non-active subtrees

### DISCUSSION THREAD

**Discussion by nemec** [TriliumNext/Trilium#1612] [Q&A] [answered]
I've noticed that after some time subtrees that I am not actively viewing are auto-collapsed up to the root. Is this functionality able to be disabled? I see in the following Issue the auto-collapse feature was suggested as a way to reduce speed issues, but I don't have enough notes (yet) for that to be an issue and it's nice to have quick access to certain subtrees even if I only use them once per day.

https://github.com/zadam/trilium/issues/1192#issuecomment-674945123
> As a mitigation I'm thinking about "auto collapsing" - that nodes would automatically collapse after a period of inactivity (and if they are not opened in any of the tabs).

**Comment by zadam** [comment, marked as answer]
Hi, it is not possible to disable this ATM.

Performance was perhaps the motivating factor, but probably more importantly it keeps the note tree mentally manageable. Once you open (and not close) many subtrees then it becomes really unwieldy.

>  it's nice to have quick access to certain subtrees even if I only use them once per day.

This is of course a valid need, but I would tend to address it differently. For 0.47 I plan some kind of bookmark system which would allow quick access.

In the mean time you can open the note in a tab which keeps it from getting closed and on its own works as a quick access shortcut ...

**Reply by nemec** [reply]
> In the mean time you can open the note in a tab

Thanks, I think this will be enough for my needs.

Best of both worlds would be the ability to pin a specific note from having its children collapsed from the right-click menu, but that's probably too much effort for little value.
